src/fs-poll.c

```diff
@@ -77,7 +77,7 @@ int uv_fs_poll_start(uv_fs_poll_t* handle,
 
   loop = handle->loop;
   len = strlen(path);
-  ctx = uv__calloc(1, sizeof(*ctx) + len);
+  ctx = (struct poll_ctx *)uv__calloc(1, sizeof(*ctx) + len);
```

Comment:
If this lands, all of the pointers should be updated so that the star leans left. For example, `(struct poll_ctx*)` here.